components/omega/doc/design/VertCoord.md

```diff
@@ -0,0 +1,264 @@
+(omega-design-vert-coord)=
+# Vertical Coordinate
+
+## 1 Overview
+The vertical coordinate module will be responsible for computing and storing information related to the vertical mesh in Omega.
+Since Omega will be a non-Boussinesq model, the vertical independent variable will be pressure- as opposed to $z$-based.
```

Comment:
```suggestion
Since Omega will be a non-Boussinesq model, the vertical independent variable will be pressure-based (expressed in terms of the pseudo-height) as opposed to $z$-based.
```